pkg/resource/aws/testdata/acc/aws_db_subnet_group/terraform.tf

```diff
@@ -8,32 +8,52 @@ terraform {
   }
 }
 
+locals {
+    timestamp = formatdate("YYYYMMDDhhmmss", timestamp())
+    prefix = "vpctest-${local.timestamp}"
```

Comment:
`s/vpctest/aws_db_subnet_group/` will help us to associate theses resources to a test case.